src/types/definition.ts

```diff
@@ -46,7 +46,7 @@ export interface IFood {
   name: string;
   quantity: number;
```

Comment:
Based on the document, is quantity also supposed to be nullable? 
<img width="308" alt="Screenshot 2024-02-07 at 23 15 29" src="https://github.com/nishiki-tech/nishiki-frontend/assets/99339182/a8f4ce0d-6fac-48b7-908e-7b2b76f7045d">
Correct me if I'm missing some context on issue or something 🙏

Reply:
@kanta1207
I apologize for my oversight. I have corrected the mistake, please review it again.